sidebars.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     {
       "Welcome": [
       "overview",
+      "saas-vs-self-hosted",
```

Comment:
Not sure if this is the right place. The user doesn't know about the Okteto CLI. Should we have an intro about Okteto CLI and Okteto Platform in the "overview" page?

Reply:
Yes, I think that would make sense. Part of placing this here is to experiment with the overall navigation hierarchy and structure of the docs.

My general thought is:

1. Educate the user about high-level concepts (What Okteto is, CLI, Platform, SaaS vs. Self-Hosted)
2. Dive into specific topics for each
3. Provide/keep the references/tutorials/etc sections

Since many features overlap between CLI and Platform it may not work with this proposal, but at a minimum I think it makes sense to frontload the topics in point `1` 👍